frontend/src/components/requestChanges/ReportChanges.vue

```diff
@@ -12,8 +12,8 @@
       <v-container>
         <p class="text-h6 text-center">What changes do you want to make?</p>
         <v-row>
-          <v-col cols="12" md="6" xl="4">
-            <SmallCard v-if="organizationProviderType == 'GROUP'">
+          <v-col v-if="organizationProviderType == 'GROUP'" cols="12" md="6" xl="4">
```

Comment:
This should be ===.

Reply:
done! I also fixed the similar issues in this file as well